### PR TITLE
basic VCS flow setup for cv32e40p/release

### PR DESCRIPTION
**Pull request by abdullahyildiz**

### CODE REVIEW

File: cv32e40p/sim/core/Makefile
Comment (MikeOpenHWGroup) [inline cv32e40p/sim/core/Makefile:124]: I know they did not before this pull-request, but these variables should use conditional assignment:
```
VCS_VERSION ?= Q-2020.03-SP2
VCS_HOME      ?= /eda/synopsys/vcs/$(VCS_VERSION)
```
This will allow teams with differing installations to use this Makefile without modification (the variables can be set as shell variables).

File: cv32e40p/README.md
Comment (MikeOpenHWGroup) [inline cv32e40p/README.md:17]: This is redundant.  The proper place to put this is [$(CORE_V_VERIF)/mk/README.md](https://github.com/openhwgroup/core-v-verif/tree/cv32e40p/release/mk#running-the-environment-with-synopsys-vcs-vcs-vcs).  Look for "Running the environment with Synopsys VCS".

File: cv32e40p/README.md
Comment (MikeOpenHWGroup) [inline cv32e40p/README.md:21]: This works for your environment, but will not work for everyone.   There are already detailed instructions for this in the `mk` directory's [README](https://github.com/openhwgroup/core-v-verif/blob/cv32e40p/release/mk/README.md) and [TOOLCHAIN](https://github.com/openhwgroup/core-v-verif/blob/cv32e40p/release/mk/TOOLCHAIN.md) docs.

**Review by MikeOpenHWGroup — CHANGES_REQUESTED** [review]
This is great @abdullahyildiz, thanks for your contribution!   Of course, it is never that easy and I have requested a couple of changes.

Also, can you direct this PR to `cv32e40p/dev`?   The `release` branches are for staging merges from `master` and so we typically do not merge pull-requests directly to `release`.

File: cv32e40p/README.md
Comment (abdullahyildiz) [reply cv32e40p/README.md:17]: Please ignore this commit.

File: cv32e40p/README.md
Comment (abdullahyildiz) [reply cv32e40p/README.md:21]: Please ignore this commit.

**Comment by abdullahyildiz** [comment]
@MikeOpenHWGroup,

Could you selectively approve the PR? If not, I should modify the PR to create a new branch.

**Comment by MikeOpenHWGroup** [comment]
> Could you selectively approve the PR? If not, I should modify the PR to create a new branch.

Sorry, I cannot approve this PR because it must point to the `cv32e40p/dev` branch.   Maybe the easiest thing for you is to close this PR and create a new one.

**Pull request closed without merging**
